ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -1,16 +1,16 @@
 base_images:
+  go-builder:
```

Comment:
We want all those changes in the relelase-1.0 branch as well I guess?

Reply:
We do indeed. Added them to `release-1.0`.